modules/samples/producers/video-frame-client/pom.xml

```diff
@@ -0,0 +1,34 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Fixed in https://github.com/hemikak/product-cep/commit/6210d26b510acc5fc699d74e24cdcd17d3c8dd1d